From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Thu, 16 May 2024 17:17:50 -0400
Subject: Always build common/status-codes.h and common/audit-events.h

This reduces our dependence on files injected into the upstream
tarball.

This addresses part of #1071202

Forwarded: not-needed
---
 common/Makefile.am | 2 --
 1 file changed, 2 deletions(-)

diff --git a/common/Makefile.am b/common/Makefile.am
index f55a9c3..e5260c2 100644
--- a/common/Makefile.am
+++ b/common/Makefile.am
@@ -140,7 +140,6 @@ endif
 libgpgrl_a_SOURCES = \
         gpgrlhelp.c
 
-if MAINTAINER_MODE
 # Note: Due to the dependency on Makefile, the file will always be
 # rebuilt, so we allow this only in maintainer mode.
 
@@ -159,7 +158,6 @@ status-codes.h: Makefile.am mkstrtable.awk exstatus.awk status.h
 	$(AWK) -f $(srcdir)/exstatus.awk $(srcdir)/status.h \
 	  | $(AWK) -f $(srcdir)/mkstrtable.awk -v textidx=3 -v nogettext=1 \
 		   -v pkg_namespace=statusstr_  > $(srcdir)/status-codes.h
-endif
 
 #
 # Module tests
